group_name: "cartesian_arm"
start_configuration: [0.01,0.01,0.01]
goal_configuration:  [0.9,0.9,0.9]
utopia_tolerance: 0.05
extend: false
max_distance: 0.1
use_kdtree: true
checker_resolution: 0.01
parallel_checker_n_threads: 4
